config/locales/ja/manageiq.po

```diff
@@ -6558,7 +6558,7 @@ msgstr "派生 VM 数 (オフ)"
 msgid "MetricRollup|Derived vm count on"
 msgstr "派生 VM 数 (オン)"
 
-msgid "MetricRollup|Derived vm numvcpus"
+msgid "MetricRollup|Derived vm cpu_sockets"
```

Comment:
Don't change the .po files.  There are rake tasks to do it automatically and aid translators in knowing which values have changed.